Feature/end attempt interaction

ssipasseuth opened the pull request:
Add the standard end attempt interaction in TAO

krampstudio commented on views/js/qtiCommonRenderer/renderers/interactions/EndAttemptInteraction.js:
```diff
@@ -0,0 +1,125 @@
+        //on click, 
+        $container.on('click.commonRenderer', function(){
+            $container.val(true);
+            console.log('trigger submitting', getResponse(interaction));
```
Please remove `console.log`

krampstudio commented on views/js/qtiCommonRenderer/renderers/interactions/EndAttemptInteraction.js:
```diff
@@ -0,0 +1,125 @@
+    'taoQtiItem/qtiCommonRenderer/helpers/Helper',
+    'taoQtiItem/qtiCommonRenderer/helpers/PciResponse',
+    'i18n'
+], function(_, tpl, Helper, pciResponse, __){
```
Add the `"use strict";` declaration after the main function (same for the other files)

krampstudio commented on views/js/qtiCreator/helper/creatorRenderer.js:
```diff
@@ -9,6 +9,19 @@ define([
     //configure and instanciate once only:
     var _creatorRenderer = null;
     
+    var _configurableInteractions = ['endAttempt'];
```
Please comment this section/function so others know what it is used for

krampstudio commented:
Some files also miss comments/jsdoc

krampstudio commented:
Let me know when it's ready to merge

ssipasseuth commented:
Oh sorry, forgot to tell you. 
Yes, it is ready now : I added some jsdoc and added "use strict" on a bunch of files for the item creator source (not every one though). I will do it progressively as I see them in the future.
